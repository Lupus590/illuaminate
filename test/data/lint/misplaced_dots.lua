-- config: (lint (only (syntax:misplaced-dots)))

local function f(..., x) end
local function f(x, ...) end -- ok
