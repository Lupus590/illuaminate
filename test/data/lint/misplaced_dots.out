misplaced_dots.lua:[3:17-3:20]: Varargs can only appear as the last argument to a function. [syntax:misplaced-dots]
   │
 3 │ local function f(..., x) end
   │                  ^^^
Compilation failed with 1 errors and 0 warnings
